middlewares/accesslog/logger_test.go

```diff
@@ -68,10 +69,55 @@ func TestLogger(t *testing.T) {
 
 	logger.ServeHTTP(&logtestResponseWriter{}, req, LogWriterTestHandlerFunc)
 
-	if logdata, err := ioutil.ReadFile(logfilePath); err != nil {
+	if logdata, err := ioutil.ReadFile(logfilePath); err == nil {
+		assertValidLogData(t, logdata)
+	} else {
 		fmt.Printf("%s\n%s\n", string(logdata), err.Error())
```

Comment:
I compared the messages a bit with and without. I dropped it then completely as it doesn't really add value. The logdata will always be empty given there is an error and err infos we get anyway using `require.NoError`.